implementations/rust/ockam/ockam_node_test_attribute/tests/trybuild.rs

```diff
@@ -0,0 +1,7 @@
+#[test]
+fn trybuild() {
+    let t = trybuild::TestCases::new();
+    t.pass("tests/node/pass*.rs");
+    t.compile_fail("tests/node/fail*.rs");
+    t.compile_fail("tests/node_test/*.rs");
```

Comment:
Maybe worth adding a comment why tests that start with "pass_" goes under compile_fail all here. Wasn't obvious for me

Reply:
100% agree to add it here too (I initially added this to the PR description)